docs/source/place_and_route.rst

```diff
@@ -0,0 +1,317 @@
+Place and Route
+===============
+
+Rig provides a set of algorithms and utilities for mapping graph-like
+applications onto specific cores in a SpiNNaker machine and defining routes
+between them. Broadly, the task is split into three steps:
+
+* **Placement**: Assign graph vertices to a chips.
+* **Allocation**: Allocate specific chip resources to each vertex (e.g. cores,
+  memory).
+* **Routing**: Generate routes to connect vertices according to a supplied set
+  of nets.
+
+Rig provides a selection of complementary algorithm implementations for each
+step which attempt to carry out these tasks. Users are, of course, free to use
+their own application-specific processes in place of any or all of these steps.
+
+Terminology
+-----------
+
+The key pieces of terminology used are defined below:
+
+Application Graph
+    The `hyper-graph <http://en.wikipedia.org/wiki/Hypergraph>`_ which
+    describes how an application's computational resources (the *vertices*) are
+    connected to each other by *nets*.
+Vertex
+    A *vertex* in *application graph*. Each vertex is mapped onto exactly one
```

Comment:
Er, maybe not.  I thought it odd that vertex was used in a sentence without being defined.